lib/web/apiserver.go

```diff
@@ -806,6 +806,8 @@ func (h *Handler) bindDefaultEndpoints() {
 	h.GET("/webapi/mfa/devices", h.WithAuth(h.getMFADevicesHandle))
 	h.POST("/webapi/mfa/authenticatechallenge", h.WithAuth(h.createAuthenticateChallengeHandle))
 	h.POST("/webapi/mfa/devices", h.WithAuth(h.addMFADeviceHandle))
+	// DEPRECATED in favor of mfa/authenticatechallenge.
+	// TODO(bl-nero): Remove in v17.
```

Comment:
We tend to use the format `TODO(username): DELETE IN 17.0.0`. I know that Zac and Roman sometimes grep for this string and ask people about potential cleanups in an upcoming major version.